app/shared/src/popup/BarPopup.js

```diff
@@ -15,6 +15,20 @@ type Props = {|
   onSave: Function,
 |};
 
+const saveLinkButtonStyles = {
```

Comment:
I would prefer `StyleSheet.create`. Also, we should not reinvent a wheel here so this may be better (`LinkButton`):

```diff
- styles={{ ...styles, ...buttonProps.styles }}
+ styles={[ styles, buttonProps.styles ]}
```

Reply:
This will not work with how the button component is now